agreed/0009-MPID-Messaging/0009-MPID-Messaging.md

```diff
@@ -533,17 +520,12 @@ impl MpidManager {
         }
     }
 
-    fn get_message(header: (sender_name: ::routing::NameType,
+    fn get_message(header: (sender_name: XorName,
                             sender_public_key: ::sodiumoxide::crypto::sign::PublicKey,
                             mpid_header: MpidHeader)) {
-        let request_sd = StructuredData {
-            type_tag: MPID_MESSAGE,
-            identifier: header.sender_name,
-            data: ::utils::encode(MpidMessgeWrapper::GetMessage(mpid_header_name(mpid_header))),
-            previous_owner_keys: vec![],
-            version: 0,
-            current_owner_keys: vec![header.sender_public_key],
-            previous_owner_signatures: vec![]
+        let request_sd = PlainData {
+            name: mpid_header.sender_name,
+            value: seialise(MpidMessgeWrapper::GetMessage(mpid_header_name(mpid_header))),
```

Comment:
referring to the implementation PR https://github.com/maidsafe/safe_vault/pull/317
it turns out the MpidMessageWrapper only needs to be serving as content wrapper, holding MpidHeader or MpidMessage .
I remember the previous enum is to serving as a operation tag in SD.